src/schneider_lab_to_nwb/schneider_2024/schneider_2024_behaviorinterface.py

```diff
@@ -1,25 +1,109 @@
 """Primary class for converting experiment-specific behavior."""
 from pynwb.file import NWBFile
+from pydantic import FilePath
+import numpy as np
+from h5py import File
```

Comment:
Do you have all the data already? 

Whether something in matlab is encoded as hdf5 or matlab native format depends on some run time trick that people do when they save the file. In my experience, it is the case that for other conversions some files are hdf5 and some are not. 

If you are sure that all of them are hdf5 encoded this is not a problem. If you are not, I would not bother checking and instead use a robust solution that can read both cases like pymatreader.

Reply:
> Do you have all the data already?

No, I only have this one session.  Thanks for the heads up, I will be sure to check out pymatreader if this comes up as an issue.